### PR TITLE
Improve tooling

### PR DESCRIPTION
**Pull request by rosborne132**
**CHANGELOG** :memo:

* Improve tooling by linting all files on every commit.
* Improve tooling by running all tests on every push.

**PRINTS(OPTIONAL)** :framed_picture:

**LABELS**

* enhancement
* hacktoberfest

### CODE REVIEW

**Comment by pyymenta** [comment]
@rosborne132 Thank you very much! I'll review it soon!

**Comment by rosborne132** [comment]
@pyymenta what is the progress on this review? :)

File: package.json [hunk 1]
```diff
@@ -36,16 +41,25 @@
...
+        ]
+    },
+    "husky": {
+        "hooks": {
+            "pre-commit": "lint-staged",
+            "pre-push": "yarn test"
+        }
     }
-}
+}
```
Comment (pyymenta) [inline package.json:65]: EOF here

**Review by pyymenta — APPROVED** [review]
Looks good to me @rosborne132 

We have some conflicts with the master branch.. Could you fix it?

Thank you!

**Comment by rosborne132** [comment]
@pyymenta Changes have been merged! :)

**Review by pyymenta — APPROVED** [review]
Alright! Thank you!

**Pull request merged**